api/src/pcapi/validation/models/venue.py

```diff
@@ -13,7 +13,7 @@ def validate(venue: Venue, api_errors: ApiErrors) -> ApiErrors:
 
     if venue.managingOffererId:
         if not venue.managingOfferer:
-            managing_offerer = offerers_models.Offerer.query.get(venue.managingOffererId)
+            managing_offerer = offerers_models.Offerer.query.filter_by(id=venue.managingOffererId).one()
```

Comment:
Cette modification et la modification similaire dans `tests/conftest.py` sont dans le commit "Do not rely on SQLA1.4 implicit behavior of adding to session relationships". Elles devraient être dans le commit "fix LegacyAPIWarning: The Query.get() method is considered legacy".

Si c'est trop lourd à déplacer dans le bon commit, je survivrai. ;)